src/status_im2/contexts/quo_preview/standart_auth/keycard_locked.cljs

```diff
@@ -0,0 +1,24 @@
+(ns status-im2.contexts.quo-preview.standart-auth.keycard-locked
+  (:require [quo2.core :as quo]
+            [reagent.core :as reagent]
+            [status-im2.contexts.quo-preview.preview :as preview]
+            [utils.re-frame :as rf]
+            [status-im2.common.standard-auth.keycard.view :as standard-auth]))
+
+(def descriptor
+  [{:key     :theme
+    :type    :select
+    :options [{:key :dark}
+              {:key :light}
+              {:key nil :value "System"}]}])
+
+(defn preview
```

Comment:
Please, rename the preview var to just `view` (same for the other preview namespaces). That's the convention we're trying to enforce when view namespaces expose only one public Reagent component.